src/templates/pages/index/data.json

```diff
@@ -1,4 +1,504 @@
+      "link": "#"
+    }
+  ],
+  "banner": {
+    "title1": "with",
+    "span": "binabox",
+    "title2": "explore nft collection",
+    "subtitle": "We are the best way to check rarity of NFT collection",
+    "stats": [
+      {
+        "term": "2240+",
+        "desc": "Total items"
+      },
+      {
+        "term": "1000+",
+        "desc": "Profiles whitelisted"
+      }
+    ]
+  },
+  "bannerButton": [
+    {
+      "link": "#",
+      "label": "Connect Wallet"
+    },
+    {
+      "link": "#",
+      "label": "Whitelist now"
+    }
+  ],
+  "catalog": {
+    "title": "Catalog",
+    "button": [
+      {
+        "link": "#",
+        "label": "See All"
+      }
+    ],
+    "arrow": [
+      {
+        "path": "img/images/catalog/arrow-left.svg",
+        "desc": "Arrow pointing to the left"
+      },
+      {
+        "path": "img/images/catalog/arrow-right.svg",
+        "desc": "Arrow pointing to the right"
+      }
+    ],
+    "products": [
+      {
+        "label": "New",
+        "image": "img/images/catalog/skeleton-2x.webp",
+        "alt": "A stylized skeletal figure on a dark teal background",
+        "price": "$199",
+        "name": "Name product"
+      },
+      {
+        "label": "New",
+        "image": "img/images/catalog/cat-2x.webp",
+        "alt": "A black cat with glowiteal eyes, set against a dark background",
+        "price": "$199",
+        "name": "Name product"
+      },
+      {
+        "label": "Sold out",
+        "image": "img/images/catalog/scissors-2x.webp",
+        "alt": "A pair of teal-toned scissors on a dark background",
+        "price": "$199",
+        "name": "Name product"
+      },
+      {
+        "label": "New",
+        "image": "img/images/catalog/neon-card-2x.webp",
+        "alt": "A glowing teal rectangular card design against a dark background",
+        "price": "$199",
+        "name": "Name product"
+      },
+      {
+        "label": "New",
+        "image": "img/images/catalog/neon-cube-2x.webp",
+        "alt": "A transparent cube with bright teal circuitry on a dark background",
+        "price": "$199",
+        "name": "Name product"
+      }
+    ]
+  },
+  "blog": {
+    "title": "Our blog",
+    "blogCards": [
+      {
+        "label": "NFT",
+        "image": "img/images/our-blog/face-sculpture-2x.webp",
+        "alt": "Surreal woman's face emerging from swirling water",
+        "writer": "Tony Nguyen",
+        "date": "20 Jun 2022",
+        "desc": "One September 22, 2021 A volutpat nulla eros, nulla ultrices necnulla imperdiet..."
+      },
+      {
+        "label": "NFT",
+        "image": "img/images/catalog/skeleton-2x.webp",
+        "alt": "Neon glowing skull against a dark background",
+        "writer": "Tony Nguyen",
+        "date": "20 Jun 2022",
+        "desc": "One September 22, 2021 A volutpat nulla eros, nulla ultrices necnulla imperdiet..."
+      },
+      {
+        "label": "NFT",
+        "image": "img/images/catalog/cat-2x.webp",
```

Comment:
I see here image `cat-2x`, do we have basic `cat`? 🐈 
If we need it, i suggest do two fields for images:
```
image: 'path to image',
image2: 'path to image x2'
```